platform/administer/shared-database/shared-database.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Shared database vCluster Platform
+sidebar_label: Shared Database
+sidebar_position: 1
```

Comment:
Could you also add `description` to the front matter. This helps discover the docs by search engines.

```suggestion
sidebar_position: 1
description: Learn how to use shared database with vcluster.
```

Reply:
Done, I took the description you provided :P.